doc/dev/vstart-runner.rst

```diff
@@ -0,0 +1,81 @@
+Running Python tests locally
+===============================
+
+Running your first test
+--------------------------
+Python tests in Ceph repo can be executed on the local machine using
+`vstart_runner.py`_. To do that you'd need `teuthology`_
+installed on your machine -
+
+::
+    $ git clone https://github.com/ceph/teuthology
+    $ cd teuthology/
+    $ virtualenv ./venv
+    $ source venv/bin/activate
+    $ pip install --upgrade pip
+    $ pip install -r requirements.rxt
```

Comment:
There is a typo here:
```suggestion
    $ pip install -r requirements.txt
```

Reply:
Fixed. https://github.com/ceph/ceph/pull/29907/files#diff-bb4c920ecb14ce4451cc1aa2009e5c21R16